Release build not working

Stacktrace:

<pre><code>
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck E/RClassHelper﹕ No R class found for given package name de.eidottermihi.raspicheck
    java.lang.ClassNotFoundException: de.eidottermihi.raspicheck.R
            at java.lang.Class.classForName(Native Method)
            at java.lang.Class.forName(Class.java:309)
            at java.lang.Class.forName(Class.java:273)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromPackageName(Unknown Source)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromApplication(Unknown Source)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromActivity(Unknown Source)
            at de.fhconfig.android.library.injection.xml.ActivityXmlInjector.<init>(Unknown Source)
            at de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity.onCreate(Unknown Source)
            at de.eidottermihi.rpicheck.activity.MainActivity.onCreate(Unknown Source)
            at android.app.Activity.performCreate(Activity.java:6034)
            at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2293)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2402)
            at android.app.ActivityThread.access$900(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1318)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5273)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "de.eidottermihi.raspicheck.R" on path: DexPathList[[zip file "/data/app/de.eidottermihi.raspicheck-2/base.apk"],nativeLibraryDirectories=[/vendor/lib, /system/lib]]
            at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
            at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
            at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
            at java.lang.Class.classForName(Native Method)
            at java.lang.Class.forName(Class.java:309)
            at java.lang.Class.forName(Class.java:273)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromPackageName(Unknown Source)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromApplication(Unknown Source)
            at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromActivity(Unknown Source)
            at de.fhconfig.android.library.injection.xml.ActivityXmlInjector.<init>(Unknown Source)
            at de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity.onCreate(Unknown Source)
            at de.eidottermihi.rpicheck.activity.MainActivity.onCreate(Unknown Source)
            at android.app.Activity.performCreate(Activity.java:6034)
            at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2293)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2402)
            at android.app.ActivityThread.access$900(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1318)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5273)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
    Suppressed: java.lang.ClassNotFoundException: de.eidottermihi.raspicheck.R
            at java.lang.Class.classForName(Native Method)
            at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
            at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
            at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
            ... 23 more
     Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ java.lang.ClassNotFoundException: de.eidottermihi.raspicheck.R
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.Class.classForName(Native Method)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.Class.forName(Class.java:309)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.Class.forName(Class.java:273)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromPackageName(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromApplication(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.fhconfig.android.library.injection.helper.RClassHelper.getRClassFromActivity(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.fhconfig.android.library.injection.xml.ActivityXmlInjector.<init>(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity.onCreate(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at de.eidottermihi.rpicheck.activity.MainActivity.onCreate(Unknown Source)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.Activity.performCreate(Activity.java:6034)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2293)
11-05 22:25:15.648  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2402)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.ActivityThread.access$900(ActivityThread.java:151)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1318)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.os.Handler.dispatchMessage(Handler.java:102)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.os.Looper.loop(Looper.java:135)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at android.app.ActivityThread.main(ActivityThread.java:5273)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.reflect.Method.invoke(Native Method)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.reflect.Method.invoke(Method.java:372)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ Caused by: java.lang.ClassNotFoundException: Didn't find class "de.eidottermihi.raspicheck.R" on path: DexPathList[[zip file "/data/app/de.eidottermihi.raspicheck-2/base.apk"],nativeLibraryDirectories=[/vendor/lib, /system/lib]]
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ ... 22 more
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ Suppressed: java.lang.ClassNotFoundException: de.eidottermihi.raspicheck.R
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.Class.classForName(Native Method)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ ... 23 more
11-05 22:25:15.649  14649-14649/de.eidottermihi.raspicheck W/System.err﹕ Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available
11-05 22:25:15.822  14649-14649/de.eidottermihi.raspicheck D/AndroidRuntime﹕ Shutting down VM
11-05 22:25:15.823  14649-14649/de.eidottermihi.raspicheck E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: de.eidottermihi.raspicheck, PID: 14649
    java.lang.RuntimeException: Unable to start activity ComponentInfo{de.eidottermihi.raspicheck/de.eidottermihi.rpicheck.activity.MainActivity}: de.fhconfig.android.library.injection.exceptions.InjectionException: Could not determine which Annotation to inject.
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2340)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2402)
            at android.app.ActivityThread.access$900(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1318)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5273)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
     Caused by: de.fhconfig.android.library.injection.exceptions.InjectionException: Could not determine which Annotation to inject.
            at de.fhconfig.android.library.injection.xml.XmlInjector.injectAnnotations(Unknown Source)
            at de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity.onCreate(Unknown Source)
            at de.eidottermihi.rpicheck.activity.MainActivity.onCreate(Unknown Source)
            at android.app.Activity.performCreate(Activity.java:6034)
            at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2293)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2402)
            at android.app.ActivityThread.access$900(ActivityThread.java:151)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1318)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5273)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:905)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:700)
11-05 22:25:18.074  14649-14649/de.eidottermihi.raspicheck I/Process﹕ Sending signal. PID: 14649 SIG: 9
</code></pre>

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -54,7 +54,8 @@ android {
 dependencies {
     compile project(':libraries:ssh')
     compile project(':libraries:file-explorer')
-    compile 'de.larsgrefer:android-library-base:0.8.4'
+    compile 'com.github.freefair.android-colors:material:1.0.0'
+    compile 'com.github.freefair.android-library:injection:1.0.0-rc2'
     compile 'org.ocpsoft.prettytime:prettytime:2.1.2.Final'
     compile('com.github.tony19:logback-android-classic:1.1.1-3') {
         exclude group: 'com.google.android', module: 'android'

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -56,6 +56,8 @@ dependencies {
     compile project(':libraries:file-explorer')
     compile 'com.github.freefair.android-colors:material:1.0.0'
     compile 'com.github.freefair.android-library:injection:1.0.0-rc2'
+    compile 'com.github.freefair.android-library:appcompat-preference:1.0.0-rc2'
+    compile 'com.android.support:design:23.1.0'
     compile 'org.ocpsoft.prettytime:prettytime:2.1.2.Final'
     compile('com.github.tony19:logback-android-classic:1.1.1-3') {
         exclude group: 'com.google.android', module: 'android'

--- a/app/proguard-project.txt
+++ b/app/proguard-project.txt
@@ -49,3 +49,8 @@
 -keep interface de.eidottermihi.rpicheck.ssh** { *; }
 -keep class com.lamerman.** { *; }
 -keep interface com.lamerman.** { *; }
+
+-keepattributes Signature,*Annotation*
+-keepclassmembers class * {
+    @io.freefair.android.injection.annotation.Inject* *;
+}

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/CustomCommandActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/CustomCommandActivity.java
@@ -64,10 +64,10 @@ import de.eidottermihi.rpicheck.fragment.CommandPlaceholdersDialog.PlaceholdersD
 import de.eidottermihi.rpicheck.fragment.PassphraseDialog;
 import de.eidottermihi.rpicheck.fragment.PassphraseDialog.PassphraseDialogListener;
 import de.eidottermihi.rpicheck.fragment.RunCommandDialog;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 @XmlLayout(R.layout.activity_commands)
 @XmlMenu(R.menu.activity_commands)

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/EditRaspiActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/EditRaspiActivity.java
@@ -47,10 +47,10 @@ import de.eidottermihi.rpicheck.activity.helper.Constants;
 import de.eidottermihi.rpicheck.activity.helper.Validation;
 import de.eidottermihi.rpicheck.db.DeviceDbHelper;
 import de.eidottermihi.rpicheck.db.RaspberryDeviceBean;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 @XmlLayout(R.layout.activity_raspi_edit)
 @XmlMenu(R.menu.activity_raspi_edit)

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/MainActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/MainActivity.java
@@ -51,7 +51,6 @@ import org.slf4j.LoggerFactory;
 import java.io.File;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
-import java.security.Security;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Collection;
@@ -82,9 +81,9 @@ import de.eidottermihi.rpicheck.ssh.beans.NetworkInterfaceInformation;
 import de.eidottermihi.rpicheck.ssh.beans.ProcessBean;
 import de.eidottermihi.rpicheck.ssh.beans.WlanBean;
 import de.eidottermihi.rpicheck.ssh.impl.RaspiQueryException;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 import sheetrock.panda.changelog.ChangeLog;
 
 @XmlLayout(R.layout.activity_main)

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/NewCommandActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/NewCommandActivity.java
@@ -31,10 +31,10 @@ import de.eidottermihi.raspicheck.R;
 import de.eidottermihi.rpicheck.activity.helper.Validation;
 import de.eidottermihi.rpicheck.db.CommandBean;
 import de.eidottermihi.rpicheck.db.DeviceDbHelper;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 @XmlLayout(R.layout.activity_command_new)
 @XmlMenu(R.menu.activity_command_new)

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/NewRaspiActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/NewRaspiActivity.java
@@ -29,9 +29,9 @@ import org.slf4j.LoggerFactory;
 
 import de.eidottermihi.raspicheck.R;
 import de.eidottermihi.rpicheck.activity.helper.Validation;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 @XmlMenu(R.menu.activity_raspi_new)
 public class NewRaspiActivity extends InjectionAppCompatActivity {

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/NewRaspiAuthActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/NewRaspiAuthActivity.java
@@ -47,10 +47,10 @@ import java.io.File;
 import de.eidottermihi.raspicheck.R;
 import de.eidottermihi.rpicheck.activity.helper.Validation;
 import de.eidottermihi.rpicheck.db.DeviceDbHelper;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 @XmlLayout(R.layout.activity_raspi_new_auth)
 @XmlMenu(R.menu.activity_raspi_new_auth)

--- a/app/src/main/java/de/eidottermihi/rpicheck/activity/SettingsActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/activity/SettingsActivity.java
@@ -25,7 +25,6 @@ import android.os.Bundle;
 import android.os.Environment;
 import android.preference.Preference;
 import android.preference.Preference.OnPreferenceClickListener;
-import android.preference.PreferenceActivity;
 import android.preference.PreferenceManager;
 import android.widget.Toast;
 
@@ -38,7 +37,7 @@ import de.eidottermihi.raspicheck.BuildConfig;
 import de.eidottermihi.raspicheck.R;
 import de.eidottermihi.rpicheck.activity.helper.Constants;
 import de.eidottermihi.rpicheck.activity.helper.LoggingHelper;
-import de.fhconfig.android.library.ui.AppCompatPreferenceActivity;
+import io.freefair.android.appcompatPreference.AppCompatPreferenceActivity;
 import sheetrock.panda.changelog.ChangeLog;
 
 /**

--- a/app/src/main/java/de/eidottermihi/rpicheck/widget/OverclockingWidgetConfigureActivity.java
+++ b/app/src/main/java/de/eidottermihi/rpicheck/widget/OverclockingWidgetConfigureActivity.java
@@ -45,10 +45,10 @@ import de.eidottermihi.rpicheck.activity.NewRaspiAuthActivity;
 import de.eidottermihi.rpicheck.adapter.DeviceSpinnerAdapter;
 import de.eidottermihi.rpicheck.db.DeviceDbHelper;
 import de.eidottermihi.rpicheck.db.RaspberryDeviceBean;
-import de.fhconfig.android.library.injection.annotation.XmlLayout;
-import de.fhconfig.android.library.injection.annotation.XmlMenu;
-import de.fhconfig.android.library.injection.annotation.InjectView;
-import de.fhconfig.android.library.ui.injection.InjectionAppCompatActivity;
+import io.freefair.android.injection.annotation.InjectView;
+import io.freefair.android.injection.annotation.XmlLayout;
+import io.freefair.android.injection.annotation.XmlMenu;
+import io.freefair.android.injection.ui.InjectionAppCompatActivity;
 
 
 /**

--- a/build.gradle
+++ b/build.gradle
@@ -15,5 +15,6 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://jitpack.io" }
     }
 }

